spec/support/acceptance/start_mock_servicenow_instance.sh

```diff
@@ -20,8 +22,8 @@ id=`docker ps -q -f name=mock_servicenow_instance -f status=running`
 
 if [ ! -z "$id" ]
 then
-  echo 'Mock ServiceNow container is already running'
-  exit 0
+  echo "Killing the current mock ServiceNow container (id = ${id}) ..."
```

Comment:
This makes it so if I change the mock container's code, I won't have to ssh into the VM and restart it. Makes testing the container a bit easier.